.github/workflows/pull-request-build-connector-template.yml

```
@@ -0,0 +1,51 @@
+name: PR Build (Connector)
+
+on:
+  workflow_call:
+    inputs:
+      additional-build-flags:
+        required: false
+        type: string
+        default: ""
+      additional-test-flags:
+        required: false
+        type: string
+        default: ""
+
+
+jobs:
+    ubuntu-build:
+        name: Build on Ubuntu
+        runs-on: ubuntu-latest
+        concurrency:
+          group: ${{ github.head_ref }}-ubuntu-build
+          cancel-in-progress: true
+        steps:
+            -   name: Checkout Repository
+                uses: actions/checkout@v1
```

Let's use `actions/checkout@v3` here. 
```suggestion
                uses: actions/checkout@v3
```